Add alternate file packager that can split

vvuk opened the pull request:
Crunch functionality is untested.  Works for some large demos that I've tried here; doesn't maintain 'embed' or 'compress' functionality (though the latter might come back).

juj commented:
Hey, what was the feature here? Was it to split the single huge `.data` file to multiple separate fixed size data chunks? Is this still interesting?

vvuk commented:
It was mainly to work around a bug that had us allocate enormous buffers in
Gecko when doing XHR in order to copy the downloaded data. We fixed that
bug. But it could still be useful to allow for multiple parallel
connections for downloads, or for better resiliency in case connections get
dropped.
On Mar 18, 2015 5:03 AM, "juj" <notifications@github.com> wrote:

> Hey, what was the feature here? Was it to split the single huge .data
> file to multiple separate fixed size data chunks? Is this still interesting?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kripken/emscripten/pull/952#issuecomment-82830071>.
>

kripken commented:
This PR is quite old, closing. Feel free to reopen if still relevant.
